triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Can you please try to not shuffle all of this code around in the same PR where you are making actual changes.

Reply:
That's fair. I'll resist the urge in future and land such things in two parts.